docs/markdown/Design-rationale.md

```diff
@@ -132,7 +132,7 @@ and so on.
 
 Sometimes you just have to compile files with only given compiler
 flags and no others, or install files in weird places. The system must
-allow the user to do this if he really wants to.
+allow the user to do this if they really want to.
```

Comment:
Personally I'd elide the "if he really wants to" entirely, as it implies a kind of oh *fine*, if you must..." / unwilling compromise, but the whole point of this section is arguing that it's sensible and reasonable to sometimes need it.